packages/pass-style/src/passStyle-helpers.js

```diff
@@ -157,6 +156,7 @@ const makeConfirmTagRecord = confirmProto => {
       (!isPrimitive(tagRecord) ||
         (reject && reject`A non-object cannot be a tagRecord: ${tagRecord}`)) &&
       (isFrozen(tagRecord) ||
+        harden.isFake ||
```

Comment:
```suggestion
(harden.isFake ||
        isFrozen(tagRecord) ||
```
because that's the order you use almost every else. Please normalize everywhere this appears.